src/components/info.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import InputWithErrorInfo from './input-error-info';
+
+const INFO  = 'info';
+const NAME  = 'name';
```

Comment:
Aren't `NAME`, `PHONE`, and `EMAIL` duplication for `FIELD_NAMES` in the `user`?